### DISCUSSION TITLE
Zigbee Human Presence Sensor WZ-M100-W Spam

### DISCUSSION THREAD

**Discussion by jcastroa87** [Koenkk/zigbee2mqtt#17824] [Devices]
Hi everyone,

i have a Zigbee Human Presence Sensor (WZ-M100-W) https://www.aliexpress.com/item/1005005200170242.html i can only make it work with Zigbee2MQTT Edge, but i get spamming always, so this make the whole HA work slow.

![Zigbee2MQTT-Edge-–-Home-Assistant (3)](https://github.com/Koenkk/zigbee2mqtt/assets/44643608/5611b0e5-0b79-475e-99f4-f1e0d1f4e3fb)

My Controller is SONOFF ZB Dongle-E USB Dongle Zigbee 3.0.

Thanks everyone in advance.

**Comment by nicolinuxfr** [comment]
I have noticed the same behavior, but I'm not sure if it slows HA down. I have the same controller and I wondered it this spamming was not a cause for problems, since I encounter this bug really often :  https://github.com/Koenkk/zigbee2mqtt/discussions/18154

I guess this is not a common setup though… 🤷‍♂️

**Comment by portboy** [comment]
I must reup this topic
I have the same issue and yes, it slows down my hole network. An with around 40 payload messages the porson recognation often fails ...
Any hints or maybe a solution for this?

**Reply by lux73** [reply]
did you tried using the `debounce` Option?

https://www.zigbee2mqtt.io/guide/configuration/devices-groups.html#devices-and-groups

**Reply by portboy** [reply]
thanks!
this brings much more silence :)
```
    debounce: 2
    debounce_ignore:
      - presence
```

**Reply by montoyra** [reply]
Sorry guys, I am new with zigbee2mqtt. Where I can set this configuration?
